slim.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7-slim as runtime
+EXPOSE 8080
+
+WORKDIR /girder
+
+RUN apt update && apt install -qy gcc git
```

Comment:
I recommend adding `--no-install-recommends` and explicitly installing only the packages that you need.  Also, I've found you can remove the apt cache afterwards via `&& rm -rf /var/lib/apt/lists/*`.  It is better form to you `apt-get` rather than `apt` in scripts.

Reply:
I think you're pulling `gcc` just so you get `psutils` to build.  You can use a non-official prebuilt wheel by adding `--find-links https://girder.github.io/large_image_wheels` to the `pip install` command and then you won't need `gcc`.

Reply:
> It is better form to you apt-get rather than apt in scripts.
Why?

> rm -rf /var/lib/apt/lists/*

neat!

Reply:
apt emits `WARNING: apt does not have a stable CLI interface. Use with caution in scripts.`.

Reply:
Huh, thought they both did that. Thanks.